test/users/vass/km/array-of-records-crash-2.explicit-type.chpl

```diff
@@ -7,8 +7,10 @@ const vertex_domain = {1..N_VERTICES} ;
 
 record vertex_struct {
 
+  proc init() { }
+
   // constructor
-  proc vertex_struct(nd: domain(1)) {
+  proc init(nd: domain(1)) {
```

Comment:
Here, too - let's remove the "// constructor" comment above. It does not add value.